_config.yml

```diff
@@ -72,12 +72,13 @@ collections:
     description: "JuMP News" # The post list page content
 
 # 6. Jekyll collections settings
-#defaults:
-#  -
-#    scope:
-#      path: ""
-#    values:
-#      image: "/assets/default-social-image.png" # Default image for sharing
+defaults:
+  -
+    scope:
+      path: ""
+      types: "pages"
+    values:
+      layout: "page-with-notification"
```

Comment:
To follow up on @odow's question, could you add a short comment on how to switch the notification bar on and off?

Reply:
I figured it out. You just comment out this section.

Reply:
@mlubin @odow I've changed this now so you can just comment out the notification in the config file to turn it off. No need for that default stuff anymore